code/game/objects/items/weapons/grenades/ghettobomb.dm

```diff
@@ -82,14 +55,10 @@
 				prime()
 
 /obj/item/weapon/grenade/iedcasing/prime() //Blowing that can up
-//	update_mob()
-//	explosion(epicenter, 1, 2, 3, 3)
-	explosion(src.loc,0,0,2,2)
-	del(src)
+	update_mob()
+	explosion(src.loc,-1,-1,-1, flame_range = range)	// no explosive damage, only a large fireball.
```

Comment:
You'll want to use our old explosive values here; we don't have flame_range (at least not until we get LINDA), so this bomb will literally do nothing, hahah.